QCircLearning/utils.py

```diff
@@ -62,4 +62,5 @@ def __call__(self, current_loss):
             if self.verbose:
                 print(f"EarlyStopping counter: {self.counter}/{self.patience}")
             if self.counter >= self.patience:
-                self.early_stop = True
+                stop = True
+        return stop
```

Comment:
Directly return False here.

Reply:
This indeed seems more reasonable and simplifies the logic. I made the change accordingly. Thannnnnk U！

Reply:
Possible misplace of earlystop initialization in trainermodel: Line 91-93. Earlystop should be for every training session at each iteration, not all of them. 
Suggestion: move into iteration for-loop